package.json

```diff
@@ -5,7 +5,8 @@
     "main": "lib/run.js",
     "scripts": {
         "build": "tsc --outDir .\\lib\\ --rootDir .\\src\\",
-        "test": "jest"
+        "test": "jest",
+        "test-coverage": "jest --coverage"

```

Comment:
Is there a way to supply a format or something here?

Reply:
I don't think jest gives that option.